orchestrator.Jenkinsfile

```diff
@@ -260,3 +259,46 @@ String projectVersion(project) {
         error("Error fetching latest version for ${project}: ${e}. Please check that ${project} has a docker image tagged ${img}:latest in the docker registry.\nLatest images are pushed to the registry on master branch builds.")
     }
 }
+
+/**
+ * Provides the component versions as YAML; To be passed to the kyma installer in various jobs.
+ */
+@NonCPS
+def versionsYaml(versions) {
+    def overrides = 
+"""global:
+  kyma:
+    versions:
+      - docs: ${versions['docs']}
+      - api-controller: ${versions['components/api-controller']}
```

Comment:
That whole block looks like redundant. Let's try to solve that few clashes and generate the yaml block out of the project configuration.
The advantage is that there is one less place to configure by teams when changing components.